.packit.yaml

```
@@ -30,6 +30,8 @@ jobs:
   metadata:
     targets:
     - fedora-36-x86_64
+    fmf_url: "https://gitlab.cee.redhat.com/rolivier/leapp-packit-poc-tests"
+    fmf_ref: "main"
     use_internal_tf: True
```

```suggestion
  targets:
  - fedora-36-x86_64
  fmf_url: "https://gitlab.cee.redhat.com/rolivier/leapp-packit-poc-tests"
  fmf_ref: "main"
  use_internal_tf: True
```

`metadata` is going to be deprecated; tbf I don't even know whether we supported `fmf_*` in metadata

---

Oh, good to know! For the logs I saw earlier, it seems that it worked just fine. But I will update it to remove the `metadata`! Thanks for the heads-up.